src/lib/ui/ui.ts

```diff
@@ -288,6 +288,9 @@ export const toUi = (
 			$spaces.forEach((s, i) => spaceById.set($spaceArray[i].space_id, s));
 			spaces.swap($spaces);
 
+			const $directoriesArray = $session.guest ? [] : $session.directories;
+			$directoriesArray.forEach((d) => entityById.set(d.entity_id, writable(d)));
```

Comment:
```suggestion
			$directoriesArray.forEach((d) => entityById.set(d.entity_id, writable(d)));
```

Should probably use `updateEntity` here from `entityMutationHelpers.ts`, is identical right now but may change.